src/lib.rs

```diff
@@ -1,14 +1,17 @@
 extern crate hyper;
 extern crate url;
 extern crate json_flex;
+extern crate byteorder;
 
 mod http_request;
+mod gqtp_request;
```

Comment:
Do you want to this module to be published as soon as possible?
If so, I want to add feature attributes like as `#[cfg(feature="gqtp")]` and use it here.